### PR TITLE
Add Felix Jerome Dela Fuente details

### LINKED ISSUE #299: Add details
Adding my dev details on the Dev Board

### PR DESCRIPTION
**Pull request by felixjeromedelafuente**
Closes #299

### CODE REVIEW

**Comment by Jansalvador1445** [comment]
This is good but please rebase, it may delete some data.

File: assets/data/devs.json [hunk 1]
```diff
@@ -494,19 +501,5 @@
             "facebook": "https://facebook.com/100003776636938",
             "linkedin": "https://www.linkedin.com/in/oliver-rhyme"
         }
-    },
-    {
-        "name": "Mher John F. Tan",
```
Comment (Jansalvador1445) [inline assets/data/devs.json:499]: This one, So, please rebase.

**Comment by joshuadeguzman** [comment]
Hello @felixjeromedelafuente, 

A few things from me, kindly update the title of this PR to something like "Add Felix to Devboard". In addition, I recommend adding a description to the details, such as Closes #<issue-number>.

Thank you!

**Comment by Jansalvador1445** [comment]
Merging now

**Pull request merged**